.travis.yml

```diff
@@ -79,7 +79,11 @@ jobs:
         # install helm for helm lint
         - curl -L https://git.io/get_helm.sh
           | bash -s -- -v "${HELM_VERSION}"
+        - mkdir -p /opt/commitlint && pushd /opt/commitlint \
```

Comment:
missing `&&` between `pushd /opt/commitlint` and `npm init`, I do not think you need the `\` at the end of the line in yaml